src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
```

Comment:
You don't need to wrap the content with another `div`, you can set a `container` class for the `body` and style it as needed.